android/src/ti/airship/attributes/NamedUserAttributesEditorProxy.java

```diff
@@ -0,0 +1,38 @@
+package ti.airship.attributes;
+
+import com.urbanairship.UAirship;
+import com.urbanairship.channel.AttributeEditor;
+
+import org.appcelerator.kroll.KrollProxy;
+import org.appcelerator.kroll.annotations.Kroll;
+
+import java.util.Date;
+
+import ti.airship.AirshipTitaniumModule;
+
+@Kroll.proxy(creatableInModule = AirshipTitaniumModule.class)
+public class NamedUserAttributesEditorProxy extends KrollProxy {
+
+    private final AttributeEditor editor = UAirship.shared().getNamedUser().editAttributes();
+
+    @Kroll.method
+    public void removeAttribute(String attribute) {
+        editor.removeAttribute(attribute);
+    }
+
+    @Kroll.method
+    public void setAttribute(String attribute, Object value) {
```

Comment:
What's a @Kroll method? (*just for my knowledge, I guess it's a titanium thing)

Reply:
It exposes the method to the Javascript interface. No idea why they named it kroll

Reply:
Ok thanks. Might have found the answer 😀 [https://en.wikipedia.org/wiki/Kroll_process](url)